src/metabase/models/humanization.clj

```diff
@@ -112,8 +113,8 @@
   (re-humanize-table-and-field-names!))
 
 (defsetting ^{:added "0.28.0"} humanization-strategy
-  "Metabase can attempt to transform your table and field names into more sensible, human-readable versions, e.g.
+  (tru "Metabase can attempt to transform your table and field names into more sensible, human-readable versions, e.g.
```

Comment:
gettext needs the strings to all be on one line. Unfortunately this one is pretty long. What I would suggest is having each sentence be a `tru` call, i.e.
``` clojure
(str (tru "My first sentence.")
     (tru "My second sentence."))
```
Then each sentence will appear separately for translation and the lines won't be a long in the source code.